designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/simple/SimpleStateStatus.scala

```diff
@@ -74,50 +74,50 @@ object SimpleStateStatus {
     SimpleStateStatus.ProblemStateStatus.name -> StateDefinitionDetails(
       displayableName = "Problem",
       icon = ProblemStateStatus.icon,
-      tooltip = Some(ProblemStateStatus.defaultDescription),
-      description = Some(ProblemStateStatus.defaultDescription)
+      tooltip = ProblemStateStatus.defaultDescription,
+      description = ProblemStateStatus.defaultDescription
     ),
     SimpleStateStatus.NotDeployed.name -> StateDefinitionDetails(
       displayableName = "Not deployed",
       icon = URI.create("/assets/states/not-deployed.svg"),
-      tooltip = Some("The scenario is not deployed."),
-      description = Some("The scenario is not deployed.")
+      tooltip = "The scenario is not deployed.",
+      description = "The scenario is not deployed."
     ),
     SimpleStateStatus.DuringDeploy.name -> StateDefinitionDetails(
       displayableName = "During deploy",
-      icon = URI.create("/assets/states/deploy-running.svg"),
-      tooltip = Some("The scenario has been already started and currently is being deployed."),
-      description = Some("The scenario is being deployed.")
+      icon = URI.create("/assets/states/deploy-running-animated.svg"),
```

Comment:
Will it be animated in filter selectbox? maybe we should remove this -animated from icon when presenting it in filter?

Reply:
I think replacement should be done deeper at:

```def stateDefinitions(staticIcon: true)```